app/bmc/prj.conf

```diff
@@ -40,6 +40,10 @@ CONFIG_TT_BOOT_BANNER=y
 CONFIG_TT_BOOT_BANNER_GIT_VERSION=y
 CONFIG_TT_BOOT_BANNER_SDK_VERSION=y
 
+# Raise RTT buffer size, so entire boot log is captured even if RTT isn't
+# attached
+CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=2048
```

Comment:
Yeah... I hate wasting SRAM like this, but it's kind of a necessary evil. RTT unfortunately kind of sucks.

Next gen, let's just use a UART.

Reply:
Very much agree- I also wonder why RTT doesn't just loop? IE why can't we overwrite the start of the buffer when RTT overflows

Reply:
It does, but it's not as instantaneous as hardware